core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -1484,23 +1484,18 @@ public PushEpochResponse call(TrxRegionService instance) throws IOException {
               + new String(startKey, "UTF-8") + " endKey: " + new String(endKey, "UTF-8"));
 
        boolean loopExit = false;
```

Comment:
Change looks good. Please consider removing this variable when you have a chance.